numpy/core/arrayprint.py

```diff
@@ -1637,12 +1661,92 @@ def array_str(a, max_line_width=None, precision=None, suppress_small=None):
         a, max_line_width, precision, suppress_small)
 
 
+_FORMAT_SPEC_REGEXP = re.compile(
+    r"([ +-])?"        # sign options
+    r"(\.)?([0-9]+)?"  # field precision
+    r"(.)?"            # field type
+)
+
+def _parse_format_spec(fs):
+    """
+    Parse the format spec and returns a dictionary with options for using it with `array2string`
+    This is inspired in the [Format Specification Mini-Language](https://docs.python.org/3/library/string.html#formatspec)
```

Comment:
Reflow to 80 columns and ignore the linting error because the line with the link is too long
```suggestion
    Parse the format spec and returns a dictionary with options
    for using it with `array2string`. This is inspired in the
    [Format Specification Mini-Language](https://docs.python.org/3/library/string.html#formatspec)
```